core/extensions.js

```diff
@@ -20,9 +20,8 @@
 goog.module('Blockly.Extensions');
 goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.utils');
-
-goog.requireType('Blockly.Block');
+const Block = goog.requireType('Blockly.Block');
```

Comment:
Evidently this is causing spurious eslint errors.

Reply:
Yes, I brought this up in the team chat. I think we resolved to ignore it. It's an issue with eslint not recognizing that the value is used in the closure annotations.

PR #5063 makes this a warning instead of an error and after we clang-format every file we will revisit updating the eslint config.